idaplugin/rematch/network.py

```diff
@@ -12,8 +12,12 @@
 cookiejar = CookieJar()
 opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(cookiejar))
 
-_threadpool = QtCore.QThreadPool()
-_threadpool.setMaxThreadCount(config['network']['threadcount'])
+try:
+  _threadpool = QtCore.QThreadPool()
+  _threadpool.setMaxThreadCount(config['network']['threadcount'])
+except KeyError as err:
+ from . import config
+ config = config.Config()
```

Comment:
I still don't get this. `config['network']['threadcount']` should always exist because of `self.update(DEFAULT)`.
Where am I wrong?